docker/electron-builder/README.md

```diff
@@ -17,6 +17,9 @@ docker run --rm -ti \
 build the image like:
```

Comment:
@k9ert : This command 

```
docker run --rm -ti \
 --env-file <(env | grep -iE 'DEBUG|NODE_|ELECTRON_|YARN_|NPM_|CI|CIRCLE|TRAVIS_TAG|TRAVIS|TRAVIS_REPO_|TRAVIS_BUILD_|TRAVIS_BRANCH|TRAVIS_PULL_REQUEST_|APPVEYOR_|CSC_|GH_|GITHUB_|BT_|AWS_|STRIP|BUILD_') \
 --env ELECTRON_CACHE="/root/.cache/electron" \
 --env ELECTRON_BUILDER_CACHE="/root/.cache/electron-builder" \
 -v ${PWD}:/project \
 -v ${PWD##*/}-node-modules:/project/node_modules \
 -v ~/.cache/electron:/root/.cache/electron \
 -v ~/.cache/electron-builder:/root/.cache/electron-builder \
 registry.gitlab.com/relativisticelectron/specter-desktop/electron-builder:latest
```
returns an error for me
```
docker: open /proc/self/fd/11: no such file or directory.
See 'docker run --help'.
```

So I have no way of testing the electron builder inside docker...

Reply:
I was not able to reproduce your issue with:
* Docker version 20.10.14, build a224086
* Ubuntu 20.04.4 LTS

developing/fixing those pipelines is always a horrible pain in the ass. Ever been and i can't imagine that will ever change much. Sometimes it's worth, sometimes better (e.g. cirrus has a way to run the pipeline locally).
I usually do a mixture but often enough let the pipeline run for the 20th time rather than trying to reproduce locally (while doing something else in the meantime).

I've created a new tag and pushed it this time to my fork in order to not pollute the main project. You could potentially do that as well: https://github.com/cryptoadvance/specter-desktop/blob/master/docs/continuous-integration.md#cicd-dev-env-setup